packages/server/.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
+npm run test
```

Comment:
Opinion: I'm often an opponent of requiring tests to pass for commits especially in the current workflow we are using. Its not uncommon in a feature branch to make a test-breaking-change then later update the unit test in a different commit. Since we squash commits on merge the commits written within the branch don't have to necessarily be atomic.

Reply:
Ok. I don't have a strong opinion. I can remove this and keep .husky in .gitignore.